corehq/apps/accounting/invoice_pdf.py

```diff
@@ -144,10 +145,11 @@ def get_pdf(self):
         self.draw_from_address()
         self.draw_to_address()
         self.draw_project_name()
-        self.draw_statement_period()
+        if not self.is_prepayment:
+            self.draw_statement_period()
         self.draw_invoice_label()
         self.draw_details()
-        if not self.is_wire:
+        if (not self.is_wire) or self.is_prepayment:
```

Comment:
Parens are unnecessary since [`not` has higher precedence than `or`](https://docs.python.org/2/reference/expressions.html#operator-precedence).